gwtp-core/gwtp-dispatch-rest/src/main/java/com/gwtplatform/dispatch/rest/client/core/DefaultResponseDeserializer.java

```diff
@@ -103,11 +101,16 @@ private boolean isSuccessStatusCode(Response response) {
             Serialization serialization = findSerialization(resultClass, contentType);
 
             if (serialization != null) {
-                return deserializeValue(serialization, resultClass, contentType, response.getText());
+                try {
```

Comment:
I agree it was moved out when I was using the action as a parameter argument, but I removed it so I could move it back.